its/org.sonarlint.eclipse.its.connected.sc/src/org/sonarlint/eclipse/its/connected/sc/SonarCloudConnectedModeTest.java

```diff
@@ -195,6 +195,19 @@ public void configureServerWithTokenAndOrganization() throws InterruptedExceptio
 
     assertThat(organizationsPage.getOrganization()).isEqualTo(SONARCLOUD_ORGANIZATION_KEY);
 
+    // Check that when entering an invalid organization moving forward is blocked
+    var invalidOrganization = "thisOrganizationDoesNotExist";
+    organizationsPage.setOrganization(invalidOrganization);
+    assertThat(wizard.isNextEnabled()).isTrue();
+    wizard.next();
+    new WaitUntil(new DialogMessageIsExpected(wizard, "No organizations found for key: " + invalidOrganization));
+
```

Comment:
I would add an assert here that going to the next page is not possible:

```suggestion
assertThat(wizard.isNextEnabled()).isFalse();
```

Reply:
No, as the `Next` button is not disabled. It does not react to changes in the text input fields.

Reply:
You are right. Not that is disabled. But that it is not possible to navigate to the next page. Would that be easy to test?

Reply:
Nope. But as clicking on the button spawns the dialog message, we already test for that.